internal/testdata/snapshots/output/switches/switches.go

```diff
@@ -0,0 +1,39 @@
+  package switches
+  
+  // CustomSwitch does the things in a switch
+  type CustomSwitch struct{}
+//     ^^^^^^^^^^^^ definition sg/switches/CustomSwitch#
+//     documentation CustomSwitch does the things in a switch
+  
+  // Something does some things... and stuff
+  func (c *CustomSwitch) Something() bool { return false }
+//      ^ definition local 0
+//         ^^^^^^^^^^^^ reference sg/switches/CustomSwitch#
+//                       ^^^^^^^^^ definition sg/switches/CustomSwitch#Something().
+//                       documentation Something does some things... and stuff
+  
+  func Switch(interfaceValue interface{}) bool {
+//     ^^^^^^ definition sg/switches/Switch().
+//            ^^^^^^^^^^^^^^ definition local 1
+   switch concreteValue := interfaceValue.(type) {
+//        ^^^^^^^^^^^^^ definition local 2
+//                         ^^^^^^^^^^^^^^ reference local 1
+   case int:
+    return concreteValue*3 > 10
+//         ^^^^^^^^^^^^^ reference local 2
+//         override documentation int
```

Comment:
I'm wondering if we can avoid `override documentation` by using a new symbol with definition+reference relationships with the original symbol 🤔 It doesn't feel like the best representation to override the hover at every usage site inside the case branches.

Reply:
I see that we have a special case to prohibit local symbols for `is_definition` https://sourcegraph.com/github.com/sourcegraph/scip/-/blob/scip.proto?L239-242

We can keep `override_documentation` for now and reevaluate in the future if this encoding is a problem (it's probably fine anyways and adding a symbol isn't going to be a big win)